crates/miden-lib/src/account/components/mod.rs

```diff
@@ -81,6 +95,7 @@ pub enum WellKnownComponent {
     BasicFungibleFaucet,
     RpoFalcon512,
     RpoFalcon512Acl,
+    RpoFalconMultisig,
```

Comment:
We should probably also create the actual `RpoFalconMultisig struct (similar to how we have such structs for other components - but this would be for a follow-up PR).